src/wasm-linker.cpp

```diff
@@ -210,6 +210,15 @@ void Linker::layout() {
     auto* func = out.wasm.getFunction(name);
     func->type = ensureFunctionType(getSig(func), &out.wasm)->name;
   }
+
+  // Export malloc and free whenever availble. JavsScript version of malloc has
```

Comment:
Then what we need to do is
1. Add 'malloc' and 'free' along with '_malloc' and '_free' back to emcc.py, because the prefix _ will matter again
2. Make s2wasm read an extra setting file, because the list of exported functions can be too large to be passed as something like comma-separated command-line argument (or is it not?)
3.  Make s2wasm export functions it read from that settings file or command-line argument

Do you think we should do this rather than just exporting malloc and free for now?